src/System.IO.Compression.ZipFile/tests/System.IO.Compression.ZipFile.Tests.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{775727A6-DF41-4160-A7FD-180279A653C7}</ProjectGuid>
-    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap'">$(DefineConstants);netcoreapp</DefineConstants>
```

Comment:
For this one we used to define netcoreapp for the uap target as well we should make sure this doesn't break that (assuming it still supports that target).

Reply:
I checked this one. I'll fix the usage in the .cs file instead of overloading the #define meaning.